src/shared/components/ComponentFrame/Componentframe.tsx

```diff
@@ -4,13 +4,15 @@ import style from './ComponentFrame.module.scss';
 interface ComponentFrameProps {
   title?: string;
   children: ReactNode;
+  className?: string;
 }
 
-export default function ComponentFrame({ title, children }: ComponentFrameProps) {
+export default function ComponentFrame({ title, children, className = '' }: ComponentFrameProps) {
   return (
-    <div>
-      <h4 className={style.h4}>{title}</h4>
-      <div className={style.componentFrame}>{children}</div>
+    <div className={className} style={{ display: "flex", flexDirection: "column" }}>
+      {!!title?<h4 className={style.h4}>{title}</h4>: null}
```

Comment:
eslint error, this works correctly but our current rules enforce a more idiomatic approach, like Boolean(title)

Reply:
```suggestion
      {title ? <h4 className={style.h4}>{title}</h4>: null}
```
Eslint still gives error(redundant boolean call) with `Boolean(title) ?`.  the `--fix` option has suggested this variant.
ESLint is now happy.